pulsar-client/pom.xml

```diff
@@ -29,7 +29,7 @@
     <relativePath>..</relativePath>
   </parent>
 
-  <artifactId>pulsar-client</artifactId>
+  <artifactId>pulsar-client-original</artifactId>
```

Comment:
Shouldn't we document this change?

Reply:
There will be no change for end user. Referring to `pulsar-client` will fetch the shaded jar.